ruby/data.rb

```diff
@@ -89,8 +89,10 @@
 		e.g. setting this to /tmp/mysite will place assets at /tmp/mysite/assets
 	DESC
 	task :getassets do
-		normalise_assets(:once => true)
```

Comment:
Normalise was preventing files from being bundled with wrong permissions, which resulted in problems when restoring assets. There were a lot of problems with file permissions in the past, so unless you prove `lztrpvogq` deals with them in all use-cases (which I can't even list to you), then I'd suggest leaving that here :-)